### ISSUE TITLE
Implement Gravatar images requests

### ISSUE THREAD

**Issue by coudot** [ltb-project/white-pages#78]
It could be useful to be able to answer to Gravatar images requests: https://fr.gravatar.com/site/implement/images/

This requires to scan LDAP directory for emails, compute the HASH, then store this HASH somewhere so White Pages will be able to retrieve the email from the HASH.

**Comment by bilbo-the-hobbit** [comment]
hello @coudot,

maybe then create a plugin for fusiondirectory or add this the personnal plugin in fusiondirectory, so whitepage could read it directly ?

Cheers

**Comment by coudot** [comment]
Indeed, the HASH could be computed by Fusion Directory! Does this require a plugin or just a posthook?

**Comment by bilbo-the-hobbit** [comment]
hello @coudot,

a posthook could do the trick, but i think a gravatar plugin where you can put all the various info later put in the url could be nice.

Cheers

**Comment by coudot** [comment]
The only useful information is the email HASH, and it should be computed when the email address is modified. What other fields would you manage in the plugin?

**Comment by bilbo-the-hobbit** [comment]
@coudot,

just thinking about size and rating of image stored in fusiondirectory if we do a plugin for gravatar.

Cheers
